src/super_gradients/common/sg_loggers/deci_platform_sg_logger.py

```diff
@@ -41,9 +34,6 @@ def __init__(
         model_name: Optional[str] = None,
     ):
 
-        if _imported_deci_lab_failure is not None:
-            raise _imported_deci_lab_failure
```

Comment:
but you didn't add this `raise` in  `platform_client.register_experiment`
this will just get a weird error.
you need to add an if-raise statement in `DeciClient.register_experiment`

Reply:
If we raise an error here, wouldnt it be better ? This delegates the responsability of raising an error to the client, instead of needing to do the check every time we use any method of DeciClient.
https://github.com/Deci-AI/super-gradients/blob/538ee20307f1ebc30a68e92d336d4599f02b7a6e/src/super_gradients/common/plugins/deci_client.py#L42-L47